eessi/testsuite/tests/apps/openfoam/openfoam.py

```diff
@@ -52,8 +52,20 @@ def filter_scales_8M():
     ]
 
 
+def filter_scales_1M():
+    """
+    Filtering function for filtering scales for the OpenFOAM 8M mesh test
+    returns all scales with at least half a node.
+    """
+    return [
```

Comment:
which scales are you selecting here exactly? the formatting is a bit difficult to parse for me